src/Provider/MetadataInterface.php

```diff
@@ -13,16 +13,27 @@
 
 namespace Sonata\MediaBundle\Provider;
 
-/**
- * NEXT_MAJOR: Remove CoreBundle dependency.
- */
-interface MetadataInterface extends \Sonata\CoreBundle\Model\MetadataInterface
+use Sonata\BlockBundle\Meta\MetadataInterface as BlockMetadataInterface;
```

Comment:
This is wrong. Should be: https://github.com/sonata-project/SonataMediaBundle/blob/3.x/src/Provider/MetadataInterface.php